osf/migrations/0081_add_abstractprovider.py

```diff
@@ -0,0 +1,140 @@
+                ('share_source', models.CharField(blank=True, max_length=200, null=True)),
+                ('share_title', models.TextField(blank=True, default=b'', null=True)),
+                ('additional_providers', django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=200), blank=True, default=list, null=True, size=None)),
+                ('access_token', osf.utils.fields.EncryptedTextField(blank=True, null=True)),
+                ('preprint_word', models.CharField(choices=[(b'preprint', b'Preprint'), (b'paper', b'Paper'), (b'thesis', b'Thesis'), (b'none', b'None')], default=b'preprint', max_length=10, null=True)),
+                ('subjects_acceptable', osf.utils.datetime_aware_jsonfield.DateTimeAwareJSONField(blank=True, default=list, encoder=osf.utils.datetime_aware_jsonfield.DateTimeAwareJSONEncoder, null=True)),
+                ('default_license', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='default_license', to='osf.NodeLicense')),
+                ('licenses_acceptable', models.ManyToManyField(blank=True, related_name='licenses_acceptable', to='osf.NodeLicense')),
+            ],
+            options={
+                'abstract': False,
+            },
+            bases=(dirtyfields.dirtyfields.DirtyFieldsMixin, models.Model),
+        ),
+        migrations.RunSQL(
+            [
+                """
+                INSERT INTO osf_abstractprovider (id, created, modified, _id,
+                        reviews_workflow, reviews_comments_private, reviews_comments_anonymous, name, advisory_board, description,
+                        domain, domain_redirect_enabled, external_url, email_contact, email_support, social_twitter, social_facebook, social_instagram,
+                        footer_links, facebook_app_id, example, allow_submissions, share_publish_type, share_source, share_title, additional_providers,
+                        access_token, preprint_word, subjects_acceptable, default_license_id, type)
+                    SELECT id, created, modified, _id,
+                        reviews_workflow, reviews_comments_private, reviews_comments_anonymous, name, advisory_board, description,
+                        domain, domain_redirect_enabled, external_url, email_contact, email_support, social_twitter, social_facebook, social_instagram,
+                        footer_links, facebook_app_id, example, allow_submissions, share_publish_type, share_source, share_title, additional_providers,
+                        access_token, preprint_word, subjects_acceptable, default_license_id, 'osf.preprintprovider' as type
+                    FROM osf_preprintprovider
+                """
+            ], migrations.RunSQL.noop
+        ),
+        migrations.RunSQL(
```

Comment:
Minor: This call and the `RunSQL` call above could be combined by separating the two `INSERT` statements with a `;`.